Document all available server.ini configuration settings (#1256)
H-uru/Plasma: Docs/server.ini.md

# Server configuration

This is a complete list of all supported settings for the server.ini
configuration file. See [example_server.ini](../example_server.ini) for a more
practical example.

> The server.ini actually uses Plasma console script syntax, which is also used in

Reviewer: ```suggestion
The server.ini actually uses Plasma console commands, which are also used by
```

identifier for the shard when saving login information. That is, two shards with
the same `Server.DispName` will share their saved login info with each other.

## Server.Status

**Example:** `Server.Status "https://moula.example.net/status"`

The full URL of the server status page to be displayed in the patcher and login
dialogs. Optional - if omitted, the default status message "Welcome to URU" is
displayed forever.

## Server.Signup

**Example:** `Server.Signup "https://moula.example.net/signup"`

The URL that is opened by the "Need an account?" button in the login window.
Optional - if omitted, the button is disabled.

## Server.Port

**Example:** `Server.Port 14617`

The default server port number, used where the server address doesn't contain an
explicit port number. Optional - if omitted, defaults to the standard MOUL
server port 14617.

The value is an integer and shouldn't be quoted.

## Server.(Type).Host

**Example:**

```
Server.Gate.Host "moula.example.net"
Server.File.Host "moula.example.net"
Server.Auth.Host "moula.example.net"
```

The addresses of the gatekeeper, file, and auth servers, respectively. Each

> address may be a domain nam or a plain IP addresses, optionally including a port

Reviewer 2: domain *name*

> address may be a domain nam or a plain IP addresses, optionally including a port

Reviewer 2: ```suggestion
address may be a domain name or a plain IP addresses, optionally including a port
```
